packages/mobile/components/Drawer.svelte

```diff
@@ -88,7 +90,8 @@
         class="py-6 px-5 fixed w-full flex flex-col flex-auto overflow-hidden {fullScreen
             ? 'h-screen'
             : ''} bg-white dark:bg-gray-800 {enterFromSide ? '' : 'rounded-t-2xl'}"
-        style={enterFromSide ? `left: ${position}px;` : `bottom: ${position}px;`}
+        style="{enterFromSide ? `left: ${position}px;` : `bottom: ${position}px;`} {$isKeyboardOpen &&
+            `border-bottom: ${$keyboardHeight}px solid transparent`}"
```

Comment:
Ended up using `border-bottom` because was consistent between iOS / Android WebViews. Tested with `margin-bottom` and `padding-bottom` having different issues for both platforms.